packages/gestalt/src/Borders.css

```diff
@@ -1,43 +1,48 @@
 :root {
   --gestalt-border-width: 1px;
   --gestalt-border-width-lg: 2px;
-  --gestalt-border-color-gray: #767676;
-  --gestalt-border-color-darkGray: #111;
+  --gestalt-boint: 4px;
+
+  /* primary colors */
+  --gestalt-colorRed100: #e60023;
+  --gestalt-colorGray100: #efefef;
+  --gestalt-colorGray200: #767676;
+  --gestalt-colorGray300: #111;
+
+  /* remanent lightGray colors */
```

Comment:
Are you ready to merge this and follow up with an update for these colors are would you rather wait until fixing these too?

Reply:
Yeah, I'll do a follow up.